doc/source/rllib/index.rst

```diff
@@ -66,103 +66,115 @@ for a few training iterations, then perform a single evaluation loop
+Click on the images below to see an example script for each of the listed features:
 
-.. .. container:: clear-both
 
-..    .. container:: buttons-float-left
+.. container:: clear-both
 
-..        .. image:: ../images/rllib/sigils/tf-and-torch.svg
-..            :width: 120
+    .. container:: buttons-float-left
 
-.. container::
+        .. image:: ../images/rllib/sigils/rllib-sigil-tf-and-torch.svg
+            :width: 100
+            :target: https://github.com/ray-project/ray/blob/master/rllib/examples/custom_tf_policy.py
 
-    The most **popular deep-learning frameworks**: PyTorch and TensorFlow
-    (tf1.x/2.x static-graph/eager/traced).
+    .. container::
 
+        The most **popular deep-learning frameworks**: `PyTorch <https://github.com/ray-project/ray/blob/master/rllib/examples/custom_torch_policy.py>`_ and `TensorFlow
+        (tf1.x/2.x static-graph/eager/traced) <https://github.com/ray-project/ray/blob/master/rllib/examples/custom_tf_policy.py>`_.
 
-.. .. container:: clear-both
 
-..    .. container:: buttons-float-left
+.. container:: clear-both
 
-..        .. image:: ../images/rllib/sigils/distributed-learning.svg
-..            :width: 120
+    .. container:: buttons-float-left
 
-.. container::
+        .. image:: ../images/rllib/sigils/rllib-sigil-distributed-learning.svg
+            :width: 100
+            :target: https://github.com/ray-project/ray/blob/master/rllib/examples/tune/framework.py
 
-    **Highly distributed learning**: Our RLlib algorithms (such as our "PPO" or "IMPALA")
-    allow you to set the ``num_workers`` config parameter, such that your workloads can run
-    on 100s of CPUs/nodes thus parallelizing and speeding up learning.
+    .. container::
 
-.. .. container:: clear-both
+        **Highly distributed learning**: Our RLlib algorithms (such as our "PPO" or "IMPALA")
+        allow you to set the ``num_workers`` config parameter, such that your workloads can run
+        on 100s of CPUs/nodes thus parallelizing and speeding up learning.
 
-..    .. container:: buttons-float-left
 
-..        .. image:: ../images/rllib/sigils/vectorized-envs.svg
-..            :width: 120
+.. container:: clear-both
 
-.. container::
+    .. container:: buttons-float-left
 
-    **Vectorized (batched) environments**: RLlib auto-vectorizes your (custom)
-    gym.Env classes such that RLlib environment workers can batch and thus
-    significantly speedup the action computing model forward passes.
+        .. image:: ../images/rllib/sigils/rllib-sigil-vector-envs.svg
+            :width: 100
+            :target: https://github.com/ray-project/ray/blob/master/rllib/examples/env_rendering_and_recording.py
 
-.. .. container:: clear-both
+    .. container::
 
-..    .. container:: buttons-float-left
+        **Vectorized (batched) and remote (parallel) environments**: RLlib auto-vectorizes
+        your ``gym.Env``s via the ``num_envs_per_worker`` config. Environment workers can
```

Comment:
This doesn't render super well -- do we need to escape the `s` somehow?
<img width="649" alt="Screen Shot 2021-11-12 at 12 16 16 AM" src="https://user-images.githubusercontent.com/1045250/141433722-4075a70d-9fdd-4b09-b9ac-40cebcdf7e6e.png">

Reply:
fixed